libraries/AC_WPNav/AC_Loiter.cpp

```diff
@@ -91,7 +91,7 @@ void AC_Loiter::init_target(const Vector3f& position)
     sanity_check_params();
 
     // initialise pos controller speed, acceleration
-    _pos_control.set_speed_xy(LOITER_VEL_CORRECTION_MAX);
```

Comment:
I'd rather see us use the existing API for max speed inside the quadplane code instead of changing AC_Loiter